src/Tribe/Main.php

```diff
@@ -1394,6 +1396,81 @@ public function fix_all_day_events( $old_value, $new_value ) {
 			$wpdb->query( $fix_end_dates );
 		}
 
+		/**
+		 * Selects events to be moved to trash or permanently deleted.
+		 *
```

Comment:
Add docblock params:
- `@since TBD` for when we added
- `@param` for all the params
- `@return` for which kind of return